src/applications/benefits-optimization-aquia/21-4192-employment-information/pages/remarks.js

```diff
@@ -15,7 +15,7 @@ export const remarksUiSchema = {
   // 'ui:description': getPageDescription,
   remarks: {
     remarks: textareaUI({
-      title: 'Additional remarks',
+      title: ' ',
```

Comment:
This is annoying I remember having to do stuff like this in the past... I hope it does not render a empty html tag. Curious what the rendered DOM looks like...